src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -60,10 +60,10 @@ public void GetAccessControl_FileInfo_InvalidArguments()
         [Fact]
         public void GetAccessControl_FileInfo_ReturnsValidObject()
         {
-            using var directory = new TempDirectory();
+            using var directory = new TempAclDirectory();
             using var file = new TempFile(Path.Combine(directory.Path, "file.txt"));
-            FileInfo fileInfo = new FileInfo(file.Path);
-            FileSecurity fileSecurity = fileInfo.GetAccessControl();
+            var fileInfo = new FileInfo(file.Path);
+            FileSecurity fileSecurity = fileInfo.GetAccessControl(AccessControlSections.Access);
```

Comment:
```suggestion
            FileSecurity fileSecurity = fileInfo.GetAccessControl();
```
The suggested GetAccesssControl overload already uses AccessControlSections.Access by default.